kernel-modules/build/Dockerfile.fc36

```diff
@@ -16,10 +18,19 @@ RUN dnf update -y || true; \
         kmod \
         wget \
         golang-go \
+        dwarves \
         pkg-config || true ; \
     mkdir -p /output; \
     dnf autoremove -y
 
+RUN git clone https://github.com/google/googletest.git && \
```

Comment:
Ditto.

Reply:
googletest is already a submodule of this repo. I am not sure what is the proper way to get it into this dockerfile.

Reply:
with `WORKDIR` and `COPY` directives

Reply:
Done